apps/components-e2e/src/components/filter-field/filter-field.e2e.ts

```diff
@@ -47,7 +47,7 @@ test('should not show a error box if there is no validator provided', async () =
 
 test('should show a error box if does not meet the validation function', async () => {
   await clickOption(3)
-    .typeText(input, 'a')
+    .typeText(input, 'a', { speed: 0.1 })
```

Comment:
This feels meticulously slow, no real user would type this slow. 
In my opinion we should try and mimic user speeds as much as possible in e2e tests, without introducing flakyness into the tests.

For the experimental release, this is still fine.